src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -19,37 +19,29 @@
 /* R3F's <Dom /> element*/
 .heatmap > div > div {
   transform: none !important;
-  pointer-events: none;
+  user-select: none; /* prevent text selection of tick labels when dragging */
 }
 
 .axisGrid {
   display: grid;
   grid-template-areas:
     'left-axis .'
     'empty bottom-axis';
-  grid-template-columns: 72px 1fr;
-  grid-template-rows: 1fr 36px;
```

Comment:
I now set these dynamically based on `AXIS_OFFSETS`. Not necessary, it just avoids potential future problems with duplicating the values.

Reply:
Yes I was worried about that. Nice :+1: